### PR TITLE
Added changelog

### PR DESCRIPTION
**Pull request by codekultur**
As discussed in the last API guild meeting and stated in https://github.com/zalando/restful-api-guidelines/issues/139 we like to introduce a changelog for API guideline changes. This is a first shot...

### CODE REVIEW

**Comment by whiskeysierra** [comment]
:+1:

**Comment by tfrauenstein** [comment]
👍   However, I propose to only log major changes: 
Non major changes are editorial-only changes or minor changes of existing guidelines, e.g. adding new error code. Major changes are changes that come with additional obligations, or even change an existing guideline obligation. The latter changes should be additionally labelled, for instance, with "Rule Change".  Pls. add this hints, thx.

**Comment by ePaul** [comment]
I agree to @tfrauenstein about the fact that we only want major changed. For people who want to know all details, we should add a link to https://github.com/zalando/restful-api-guidelines/commits/master.

Possible wording:

> This change log only contains major changes made after October 2016.
Non-major changes are editorial-only changes or minor changes of existing guidelines, e.g. adding new error code. Major changes are changes that come with additional obligations, or even change an existing guideline obligation. The latter changes are additionally labelled with "Rule Change" here.
To see a list of all changes, please have a look at the [commit list in Github](https://github.com/zalando/restful-api-guidelines/commits/master).

(Mostly copied from Thomas' proposal, with minor rewording and two additional sentences.)

In CONTRIBUTIONS.md we should possibly add a sentence like "When your pull request does major changes, please also add an entry to the change log."

**Comment by codekultur** [comment]
Thanks for your contributions. I like the idea to distinguish between editorial (minor) changes and rule changes and only record the later ones in the changelog. Updated the PR accordingly...

**Comment by tfrauenstein** [comment]
👍

**Comment by tlossen** [comment]
👍

File: SUMMARY.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Summary
 
+* [Introduction](README.md)
```
Comment (ePaul) [inline SUMMARY.md]: Hmm, we have Introduction.md currently. I think there is no reason for README.md to be included here. The README is more about the repository, not part of the produced document.

Comment (ePaul) [inline SUMMARY.md]: This also seems to be the reason for failing the CI build – README.md has an image which can't be included in the PDF.

**Comment by tfrauenstein** [comment]
👍

**Comment by tkrop** [comment]
:+1:

**Comment by dehora** [comment]
:+1:

**Pull request merged**